src/MongoDB.Analyzer/Core/Linq/LinqAnalysisConstants.cs

```diff
@@ -25,3 +25,9 @@ internal static class LinqAnalysisErrorMessages
 {
     public const string MethodInvocationNotSupported = "Method referencing lambda parameter is not supported LINQ expression.";
 }
+
+internal static class EFAnalysisErrorMessages
+{
+    public const string ByteArraysNotSupported = "Byte Array Property Not Supported.";
+    public const string GroupByMethodNotSupported = "GroupBy Not Supported in EF.";
```

Comment:
Not need to capitalize every word.
Consider
'Byte array type is not supported by this version of the EF provider.".
'GroupBy is not supported by this version of the EF provider.".

Reply:
Done